examples/radial_gradient_filters.py

```diff
@@ -0,0 +1,57 @@
+"""
+=======================
+Radial Gradient Filters
+=======================
+
+This example applies both the normalizing radial gradient (`sunkit_image.radial.nrgf`) filter and Fourier
+normalizing radial gradient filter (`sunkit_image.radial.fnrgf`) to a sunpy map.
+
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.map
+import sunpy.data.sample
+
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+###########################################################################
+
+# Sunpy's sample data contain a number of suitable FITS files for this purpose.
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###########################################################################
+
+# Radial bins are calculated over which the filters will be applied. The distance
+# between 1 Solar radius and 2 Solar radius is divided into 100 equal parts.
+radial_bin_edges = equally_spaced_bins()
+radial_bin_edges *= u.R_sun
+
+# The NRGF filter is applied after it.
+out1 = radial.nrgf(aia_map, radial_bin_edges)
+
+# The NRGF filtered map is plotted. 
+out1.plot()
+plt.show()
```

Comment:
I think we only want one `plt.show` at the end.

Reply:
Then when the script is run only the last plot i.e. the FNRGF will be shown.

Reply:
It should plot all the images at the end.

Reply:
In the example gallery, it will show both the plots but when suppose the python file or jupyter file is downloaded then only the last one will be plotted

Reply:
If that's the case, you will want to work out why and make sure that both plots are outputted when you run it locally.